libcob/intrinsic.c

```diff
@@ -1837,7 +1837,7 @@ cob_intr_numval (cob_field *srcfield)
 	cob_field	field;
 	unsigned char	integer_buff[64];
 	unsigned char	decimal_buff[64];
-	unsigned char	final_buff[64];
+	unsigned char	final_buff[130];
```

Comment:
Even if the size of final_buff is 129, the warning messages will be solved.
Set the size of final_buff to 129.